kubernetes/helm/pinot/values.yaml

```diff
@@ -21,8 +21,8 @@
 
 image:
   repository: apachepinot/pinot
-  tag: latest-jdk11 # release-0.7.1
-  pullPolicy: IfNotPresent
+  tag: latest # release-0.7.1
+  pullPolicy: Always
```

Comment:
Why do we need to change this

Reply:
This is to let the latest image always up to date